sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -284,7 +285,7 @@ Previously, in `Microsoft.Azure.ServiceBus`, you had the below options to receiv
 
 While the first option is similar to what you would do in a non-session scenario, the second that allows you finer-grained control is very different from any other pattern used in the library.
 
-Now in `Azure.Messaging.ServiceBus`, we simplfify this by giving session variants of the same methods and classes that are available when working with queues/subscriptions that do not have sessions enabled.
+Now in `Azure.Messaging.ServiceBus`, we simplify this by giving session variants of the same methods and classes that are available when working with queues/subscriptions that do not have sessions enabled.
```

Comment:
Let's not update the migration guide until the release is out.

Reply:
> Let's not update the migration guide until the release is out.

The migration documentation should include the feature that is released. For example, I'm looking forward to this feature because it will bring the new SDK closer to the feature parity (yes, as of now there's **no** feature parity) and allow to move off the old SDK. Therefore 👎 to releasing the feature w/o documentation.

Reply:
Hey @SeanFeldman,

I believe there is a misunderstanding here...
The suggestion was not to not document the feature. It was to wait until the release date to update the migration guide.

If we update the migration guide before the feature is released, that can be misleading. You may land on the migration guide, see that this feature is now available, but not find it when downloading the latest package from nuget.

Our release cadence is monthly, which means that an update to the package will take another 3 weeks or so at a minimum. 

My recommendation was to
- not include changes to migration guide in this PR
- review this PR and get changes checked in
- release a preview in 3 weeks or so with this feature
- update the migration guide right after the release

Reply:
Created https://github.com/Azure/azure-sdk-for-net/issues/18821

Reply:
Your explanation in combination with the follow-up PR raised by @JoshLove-msft sorts it out. Thanks.